include/boost/beast/core/detail/config.hpp

```diff
@@ -101,4 +101,12 @@ namespace net = boost::asio;
 #define BOOST_BEAST_ASYNC_TPARAM2 BOOST_ASIO_COMPLETION_TOKEN_FOR(void(::boost::beast::error_code, ::std::size_t))
 #endif
 
+#ifndef BOOST_BEAST_V1_DYNAMIC_BUFFER_PROXY
+#define BOOST_BEAST_V1_DYNAMIC_BUFFER_PROXY(type) \
+    typename std::enable_if< \
```

Comment:
as long as we are fully qualifying namespaces why not `::std`?

Reply:
actually, this define can go now. The return type of dynamic_buffer is now a documented template type.